src/Dependencies/Collections/SegmentedArray`1.cs

```diff
@@ -419,6 +465,8 @@ public void Reset()
         internal static class TestAccessor
         {
             public static int SegmentSize => SegmentedArray<T>.SegmentSize;
+            public static T[][] GetSegments(SegmentedArray<T> array)
+                => array._items;
```

Comment:
This already exists as a helper method (not sure where, would need to find it)

Reply:
Found it, PrivateMarshal.AsSegments